@types/pbjs/index.d.ts

```diff
@@ -87,7 +87,7 @@ interface PrebidMediaTypes {
 
 interface PrebidBid {
 	bidder: string;
-	params?: unknown;
+	params?: Record<string, any>;
```

Comment:
Would it be possible to avoid using `any`?

```suggestion
	params?: Record<string, unknown>;
```

Reply:
Ah, was this code generated automatically?

Reply:
Oh my, this is the beauty of TS 😉 

> Things that should be easy become hard, and things that are hard become `any`
_-- David Heinemeier Hansson_